### ISSUE TITLE
Notice of diabled API

### ISSUE THREAD

**Issue by AsixJin** [forslund/spotify-skill#155]
I installed this skill via the MyCroft marketplace and recently found out the API access was disable for this skill. I think this should be removed from the marketplace and/or notated on the marketplace description so others don't install this skill until its fixed.

**Comment by goldyfruit** [comment]
https://github.com/forslund/spotify-skill/issues/152

**Comment by forslund** [comment]
I'll be adding instructions for setting up with your own api key. this week, hopefully. then I'll update it in the marketplace, but you're right it should be noticed there that there are currently issues with the skill.

**Comment by nilskretschmer** [comment]
I dont see any message on the marketplace, yet. Is it possible to use this skill with my own API key then?

**Comment by goldyfruit** [comment]
@NKnuelle yes it is, follow the instructions in issue #152 from right above.
